server/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "gltf-validation-server",
+    "description": "glTF Validator as a Language Server.",
```

Comment:
I think this should be simplified to just "glTF Language Server". hover-test shows it has uses outside of just validation. I'd like to use definitionProvider to be able to navigate a document's index values to their definitions in a context sensitive way.